superset/assets/javascripts/explore/stores/controls.jsx

```diff
@@ -955,6 +955,18 @@ export const controls = {
     description: 'Whether to include a time filter',
   },
 
+  filter_order: {
+    type: 'SelectControl',
+    multi: true,
```

Comment:
oh I see, but what if you want two different filters to be sorted on the same metric?